src/test/java/htsjdk/samtools/util/IoUtilTest.java

```diff
@@ -394,4 +398,27 @@ public void testAssertDirectoryIsWritableFile(final File file, final boolean wri
             }
         }
     }
+    @DataProvider
+    public Object[][] fofnData() throws IOException {
+        Path fofnPath1 = inMemoryfileSystem.getPath("Level1.fofn");
+        Files.copy(TEST_DATA_DIR.resolve("Level1.fofn"), fofnPath1);
+
+        Path fofnPath2 = inMemoryfileSystem.getPath("Level2.fofn");
+        Files.copy(TEST_DATA_DIR.resolve("Level2.fofn"), fofnPath2);

```

Comment:
Strings "Level1.fofn" and "Level2.fofn" can be factor out (static or within the test) to re-use here and in the returned array.

Reply:
done.